src/pages/change-how-you-travel/index.js

```diff
@@ -211,23 +211,34 @@ const ChangeHowYouTravelPage = () => (
             </h2>
             <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
               To have the most impact{" "}
-              <b>switch to cycling, public transport and walking</b>. For those
-              who must have access to a car consider sharing one with others,
+              <b>switch to cycling, public transport, and walking</b>. For those
+              who must have access to a car, consider sharing one with others,
               increasing the utility of each individual vehicle. If you
               absolutely must own your own an electric car is your best bet —
-              though it's important to remember that the manufacturing process
-              of each electric car is still responsible for emissions. Owning an
-              electric car is not as expensive as you might think and there are
-              cost savings to be made over the long-term — especially if you
-              install solar panels at your home to generate your own energy.
+              though it's important to ensure your electricity supply is as close 
+              to 100% renewable as possible. 
+            </p>
+
+            <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
+              Owning an electric car is not as expensive as you might think, 
+              as many countries offer tax rebates, free road tax, and over the 
+              long term there are savings to be made on recharging and maintenance.
               <b>
                 {" "}
                 Many governments have already committed to ban fossil fuel
-                vehicle sales
+                vehicle sales, some by 2030, others by 2050.
```

Comment:
Also some cities (like Bristol) are banning diesel vehicles from even being in the city! - might be more to do with air pollution though...

Reply:
I'll mention that when I take the next swing. "With more cities starting to impose bans..."